### PR TITLE
chore(relocation): Improve UI copy

### PR DESCRIPTION
**Pull request by azaslavsky**
Screenshot 2024-01-19 at 10 24 28
Screenshot 2024-01-19 at 10 24 38

### CODE REVIEW

File: static/app/views/relocation/getStarted.tsx [hunk 1]
```diff
@@ -52,18 +52,28 @@ function GetStarted(props: StepProps) {
...
             aria-label="region"
-            placeholder="Select Region"
+            placeholder="Select Location"
             options={regions.map(r => ({label: r.name, value: r.url}))}
             onChange={opt => setRegionUrl(opt.value)}
           />
-          {regionUrl && (
-            <p>{t('This is an important decision and cannot be changed.')}</p>
-          )}
+          <p>
```
Comment (hubertdeng123) [inline static/app/views/relocation/getStarted.tsx:64]: There seems to be a lot of spacing between this text and the Continue Button. Could we tighten that up?

Comment (azaslavsky) [reply static/app/views/relocation/getStarted.tsx:67]: Fixed, new spacing:

Screenshot 2024-01-23 at 11 58 51

File: static/app/views/relocation/relocation.tsx [hunk 2]
```diff
@@ -108,8 +108,7 @@ function RelocationOnboarding(props: Props) {
         const response = responses.flat(1);
         response.sort((a, b) => {
           return (
-            new Date(a.dateAdded || 0).getMilliseconds() -
-            new Date(b.dateAdded || 0).getMilliseconds()
+            new Date(a.dateAdded || 0).getTime() - new Date(b.dateAdded || 0).getTime()
```
Comment (azaslavsky) [inline static/app/views/relocation/relocation.tsx:111]: cc @scttcper this fixes the bug in https://github.com/getsentry/sentry/pull/63490#discussion_r1462573812

**Comment by scttcper** [comment]
can we move the continue button to the bottom right instead of the bottom left

**Comment by azaslavsky** [comment]
> can we move the continue button to the bottom right instead of the bottom left, we usually have the primary action on the right

@jmanhart thoughts?

**Review by hubertdeng123 — APPROVED** [review]

**Comment by azaslavsky** [comment]
@scttcper will merge this for now (since that is the current position of the continue button in deployed code), and will change depending on what @jmanhart says in a follow up PR if necessary.

**Pull request merged**